test-network-function/lifecycle/suite.go

```diff
@@ -434,23 +395,17 @@ func uncordonNode(node string) {
 
 // Pod antiaffinity test for all deployments
 func testPodAntiAffinity(env *config.TestEnvironment) {
-	var deployments dp.DeploymentMap
 	ginkgo.When("CNF is designed in high availability mode ", func() {
-		testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestPodHighAvailabilityBestPractices)
-		ginkgo.It(testID, func() {
-			ginkgo.By("Should set pod replica number greater than 1 and corresponding pod anti-affinity rules in deployment")
-			for _, podUnderTest := range env.PodsUnderTest {
-				podNamespace := podUnderTest.Namespace
+		ginkgo.It("Should set pod replica number greater than 1 and corresponding pod anti-affinity rules in deployment", func() {
+			if len(env.DeploymentsUnderTest) == 0 {
+				ginkgo.Fail("No test deployments found.")
```

Comment:
same here, move in before each section

Reply:
Also, bring back the changes 
``
testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestPodHighAvailabilityBestPractices)
		ginkgo.It(testID, func() {
``

Reply:
Done.